gettsim/data/e_st_abzuege.yaml

```
@@ -1,56 +1,76 @@
-kifreib:
+kifreib_s_exm:
   name:
-    de: Kinderfreibetrag (sächliches Existenzminimum des Kindes sowie Betreuungs- und Erziehungs- oder Ausbildungsbedarf)
+    de: Kinderfreibetrag (sächliches Existenzminimum des Kindes).
     en:
   description:
     de: § 32(6) EStG
     en:
   values:
     1984-01-01:
-      value: 221
+      value: 110
       note:
     1986-01-01:
-      value: 1270
+      value: 635
       note:
```

```suggestion
       note: Artikel 1 G. v. 26.06.1985 BGBl. I S. 1153. Wert in Euro. Der Betrag im Gesetz lautet 1242 DM.
```
see https://offenegesetze.de/veroeffentlichung/bgbl1/1985/34#page=1

add source (law) and DM to EUR calculation in the notes.